Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -316,11 +319,13 @@ void MainWindow::Pause()
 
 bool MainWindow::Stop()
 {
-  if (!Core::IsRunning())
+  if (!Core::IsRunning() || m_inquiry_in_progress)
```

Comment:
I'll regard this as fixed. I remember it being very easy to reproduce when I first discovered it...